### PR TITLE
docs(feat): add content on how to set an organization profile picture using Gravatar 

### LINKED ISSUE #28557: Document how to set an organization profile picture using Gravatar
### Code of Conduct

- [x] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

"[Customizing your organization's profile](https://docs.github.com/en/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile)"

### What part(s) of the article would you like to see updated?

There are two ways to set an organization profile picture:

- Upload an image.
- Enter an email address associated with a Gravatar, in the field labeled "Gravatar email (private)." GitHub will use the Gravatar image associated with that email. This is currently not documented, and the purpose of this field isn't self-explanatory.

### Content plan

1. At the end of the article, add a new h2 heading: `## Changing your organization's profile picture`
2. In this section, add an h3 subsection: `### Uploading an image`, containing the following content.

   ```
   {% data reusables.profile.access_org %}
   {% data reusables.profile.org_settings %}
   1. Under your profile picture, click **Upload new picture**, then select an image.
   ```
3. Add another h3 subsection `### Using a Gravatar image`, containing the following content.

   ```
   {% data reusables.profile.access_org %}
   {% data reusables.profile.org_settings %}
   1. In the "Gravatar email (Private)" field, enter the email address associated with your Gravatar image.
   1. Click **Update profile**.
   ```

### Related issue

https://github.com/github/docs/issues/28553

### Additional information

The screenshot below is for reference, but does not need to be included in the article:

Screenshot of an organization settings screen in GitHub. A form field labeled "Gravatar email (private)" is highlighted, with the rest of the screen in shadow.

### Who does this affect?

Organization owners

### What is the impact to users?

Organization owners might be confused what this field is or how to use it, especially if they are not familiar with Gravatar. This question recently generated customer confusion with Support.

### PR DESCRIPTION
**Pull request by CBID2**
### Why:
This pull request adds directions on how to set an organization profile picture using Gravatar. Making this addition would help organizations who use GitHub gain a better understanding of this process. 

Closes #28557

### What's being changed (if available, include any code snippets, screenshots, or gifs):
Adds a new set of directions 

### Check off the following:

- [X] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by isaacmbrown** [comment]
Hi @CBID2! Thanks for the PR and following the instructions in the OP. Unfortunately I don't think this was the correct article to update because a team is not the same as an organization. If you hang tight I'll update https://github.com/github/docs/issues/28557 with the correct steps 🙇

**Comment by CBID2** [comment]
> Hi @CBID2! Thanks for the PR and following the instructions in the OP. Unfortunately I don't think this was the correct article to update because a team is not the same as an organization. If you hang tight I'll update #28557 with the correct steps 🙇
> 
> EDIT: Added the right steps to the OP

Made the changes @isaacmbrown! :)

File: content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md [hunk 1]
```diff
@@ -83,3 +83,20 @@ You can pin repositories that you want to feature, such as those that are freque
 1. Click **Save pins**.
 
 {% endif %}
+
+## Changing your organization's profile picture
+
```
Comment (isaacmbrown) [inline content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md:88]: It'd be good to add some introductory text here and explain the options, similar to what's in "[Personalizing your profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#changing-your-profile-picture)."

```suggestion

When you create an organization, {% data variables.product.product_name %} provides you with a randomly generated "identicon." The identicon is generated from a hash of your organization's user ID, so there's no way to control its color or pattern.

You can replace the identicon with an image that represents your organization. To replace the image, you can upload a new image or use a Gravatar image.

```

File: content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md [hunk 2]
```diff
@@ -83,3 +83,20 @@ You can pin repositories that you want to feature, such as those that are freque
...
+### Uploading an image
+
+{% data reusables.profile.access_org %}
+{% data reusables.profile.org_settings %}
+
+1. Under your profile picture, click **Upload new picture**, then select an image.
```
Comment (isaacmbrown) [inline content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md:94]: This renders fine, but for readability I'd remove the empty line 😃 

```suggestion
{% data reusables.profile.org_settings %}
1. Under your profile picture, click **Upload new picture**, then select an image.
```

**Review by isaacmbrown — COMMENTED** [review]
Thank you for the quick updates @CBID2! I left a few suggestions ✨

File: content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md [hunk 3]
```diff
@@ -83,3 +83,21 @@ You can pin repositories that you want to feature, such as those that are freque
 1. Click **Save pins**.
 
 {% endif %}
+
+## Changing your organization's profile picture
+
+When you create an organization, {% data variables.product.product_name %} provides you with a randomly generated "identicon." The identicon is generated from a hash of your organization's user ID, so there's no way to control its color or pattern.
+
+You can replace the identicon with an image that represents your organization. To replace the image, you can upload a new image or use a Gravatar image.
+### Uploading an image
```
Comment (isaacmbrown) [inline content/organizations/collaborating-with-groups-in-organizations/customizing-your-organizations-profile.md:92]: To fix the failing linter test

```suggestion
You can replace the identicon with an image that represents your organization. To replace the image, you can upload a new image or use a Gravatar image.

### Uploading an image
```

**Review by isaacmbrown — APPROVED** [review]
Thanks for the updates, looks good! 🚀

**Comment by isaacmbrown** [comment]
Hey @CBID2 Ooh cool, I am on LinkedIn `linkedin.com/in/isaac-brown-9b177519a` 🎉

**Pull request merged**